src/main/scala/org/hammerlab/guacamole/loci/partitioning/LociPartitioner.scala

```diff
@@ -56,6 +62,13 @@ trait LociPartitionerArgs
     handler = classOf[BooleanOptionHandler]
   )
   var quiet: Boolean = false
+
+  @Args4JOption(
+    name = "--allow-ungrouped-contigs",
+    usage = "Loosen requirement that regions are grouped by contigs.",
```

Comment:
Just out of curiosity, it's not obvious when / why you would or wouldn't use this. Is it just for debug? If you see a good place to document the higher level use case for this that could be useful